Mathlib/Logic/Encodable/Basic.lean

```diff
@@ -35,6 +35,8 @@ The point of asking for an explicit partial inverse `decode : ℕ → Option α`
 to make the range of `encode` decidable even when the finiteness of `α` is not.
 -/
 
+assert_not_exists Group
```

Comment:
I think we rather use `assert_not_exists Monoid` to say "don't import algebra". Can you check?

Reply:
You're right, we seem to use `Monoid` consistently. Actually applies to the other PR too, so I'll update that one as well.